### DISCUSSION TITLE
❗ Add "Toaster" to orbit

### DISCUSSION THREAD

**Discussion by alexasselin008** [gsoft-inc/sg-orbit#1123] [New components Ideas]
## Description

Add a way to display toasts to Orbit

![toasts](https://user-images.githubusercontent.com/38871812/194615888-a2c2b7f1-a12a-4d0a-a8fb-cef538718cfe.gif)

This should probably be done in a separate package

**Comment by patricklafrance** [comment]
I suggest moving this to a Github discussion.
